addons/website/static/src/snippets/s_popup/001.scss

```diff
@@ -70,3 +70,16 @@
         }
     }
 }
+
+#oe_main_menu_navbar:not(.o_hidden) + #wrapwrap main {
+    .s_popup[data-vcss='001'] {
+        .modal:not(.s_popup_top) {
+            .s_popup_size_full {
+                margin-top: $o-navbar-height;
```

Comment:
why don't we need this margin for the other options?
in any case we need a margin-top if the height of the popup exceeds the height of the page (in connected mode)

Reply:
other option doesn't need margin in case of connected because in that case, popup doesn't touch to top menu so we don't need it. height is set according to the margin. I have removed it by considering the previous fix https://github.com/odoo/odoo/commit/9cd8c15ed22e3033c654fcfa28f76b79083dcf98 but it's needed anyhow so I applied the change again.